_pages/community/events/bootcamp-2024.md

```diff
@@ -8,6 +8,17 @@ tabs:
     id: about
     active: true
     content: |
+        ---
+
+        **After the event:**
+
+        Thank you to everyone who contributed to this bootcamp and helped make it a success.
+
+        - [YouTube Videos](https://www.youtube.com/playlist?list=PL_hVbFs_loVR_8ntTTmmG6YEq3Po_4snu)
+        - [Slides](https://bootcamp.gem5.org/)
```

Comment:
Can you change this to gem5bootcamp.github.io/2024? I am envisioning that we will be updating bootcamp.gem5.org frequently. That link is going to always point to the 2024 bootcamp.